include/fmt/format-inl.h

```diff
@@ -674,7 +674,7 @@ template <int GRISU_VERSION> struct grisu_shortest_handler {
   }
 };
 
-template <typename Double, enable_if_t<sizeof(Double) == sizeof(uint64_t), int>>
```

Comment:
Can we add paretheses here and in one place below and kill the `FMT_ENABLE_IF_T` macro?

Reply:
Sure.